safe_transaction_service/history/views.py

```diff
@@ -231,17 +234,44 @@ def get_parameters(self) -> Tuple[bool, bool, bool]:
         )
         return executed, queued, trusted
 
-    def list(self, request, *args, **kwargs):
+    def get_ordering_parameter(self) -> Optional[str]:
+        return self.request.query_params.get(OrderingFilter.ordering_param)
+
+    def get_page_tx_identifiers(
+        self,
+        safe: ChecksumAddress,
+        executed: bool,
+        queued: bool,
+        trusted: bool,
+        ordering: Optional[str],
+        limit: int,
+        offset: int,
+    ) -> Optional[Response]:
+        """
+        This query will merge txs and events and will return the important
+        identifiers (``safeTxHash`` or ``txHash``) filtered
+
+        :param safe:
+        :param executed:
+        :param queued:
+        :param trusted:
+        :param ordering:
+        :param limit:
+        :param offset:
```

Comment:
I think that when somebody is going to read the code can be mistaken due he can think that this function is using ordering, limit and offset to do something, but in this case is only used to debug. 
Do you think that we could do this in another way? maybe printing this debug log, before and after `get_tx_identifiers` call? 
Other we could add a comment indicating that this three parameters are used only for debugging in this function.